fcrepo-auth-common/src/test/java/org/fcrepo/auth/common/ServletContainerAuthenticationProviderTest.java

```diff
@@ -176,13 +177,30 @@ public void testNoDelegatedAuthenticationForUsers() {
         when(delegateProvider.getDelegate(creds)).thenReturn(delegatePrincipal);
         when(delegatePrincipal.getName()).thenReturn("delegatedUserName");
 
+        // delegateProvider being HeaderProvider returns the header content in getPrincipals regardless of logged user
+        when(delegateProvider.getPrincipals(creds)).thenReturn(
+                new HashSet<>(Collections.singletonList(delegatePrincipal)));
```

Comment:
Before I merge this, we could clean it a bit. Instead of this complicated expression, how about using Guava's `Sets.newHashSet(E... elements)`?